scripts/evmos_minichef.ts

```diff
@@ -0,0 +1,72 @@
+import chai from "chai"
+import { ethers } from "hardhat"
+import { solidity } from "ethereum-waffle"
+
+const { expect } = chai
+import { MiniChefV2 } from "../build/typechain/"
+import { BIG_NUMBER_1E18 } from "../test/testUtils"
+import { deployments } from "hardhat"
+const { execute } = deployments
+
+chai.use(solidity)
+
+async function main() {
+  const { deployer } = await ethers.getNamedSigners()
+
+  const minichef: MiniChefV2 = await ethers.getContract("MiniChefV2")
+
+  // Total LM rewards is 30,000,000 but only 12,500,000 is allocated in the beginning
+  // Evmos's portion is 500_000
+  const TOTAL_LM_REWARDS = BIG_NUMBER_1E18.mul(500_000)
+  // 2 months (8 weeks)
+  const lmRewardsPerSecond = TOTAL_LM_REWARDS.div(2 * 4 * 7 * 24 * 3600)
+
+  // expect saddle per second to be 0
+  expect(await minichef.saddlePerSecond()).to.eq(0)
+  console.log("saddlePerSecond", (await minichef.saddlePerSecond()).toString())
+  console.log("totalAllocPoint", (await minichef.totalAllocPoint()).toString())
+
+  // batch transaction to set the reward rate and pool allocation points
+  const batchCall = [
+    await minichef.populateTransaction.setSaddlePerSecond(lmRewardsPerSecond),
+    await minichef.populateTransaction.set(
+      0,
+      0,
+      "0x0000000000000000000000000000000000000000",
+      false,
+    ),
+    await minichef.populateTransaction.set(
+      1,
+      100,
+      "0x0000000000000000000000000000000000000000",
+      false,
+    ),
+    await minichef.populateTransaction.updatePool(0),
```

Comment:
Should move this up to the top of the batch calls to ensure update is called first. 

nitpick: alternatively this can be `massUpdatePool([0, 1])` instead